File: BackEnd/ProjectManagerSystem/DataModelLibrary/Database/AtomEmployeeProjectStatus/DbAtomEmployeeProjectStatusEnum.cs
namespace DataModelLibrary.Database.AtomEmployeeProjectStatus;

/// <summary>資料庫-原子-員工專案狀態</summary>
public enum DbAtomEmployeeProjectStatusEnum : short
{
    /// <summary>未定義</summary>
    Undefined = 0,

    /// <summary>未指派</summary>
    NotAssigned = 1,

    /// <summary>未開始</summary>
    NotStarted = 2,

    /// <summary>如期</summary>
    OnSchedule = 3,

    /// <summary>注意</summary>
    AtRisk = 4,

    /// <summary>延遲</summary>
    Delayed = 5,

    /// <summary>已完成</summary>
    Completed = 6
}

File: FrontEnd/manager-back-site/src/constants/DbAtomEmployeeProjectStatusEnum.ts
/** 資料庫-原子-員工專案狀態 */
export enum DbAtomEmployeeProjectStatusEnum {
  /** 未定義 */
  Undefined = 0,
  /** 未指派 */
  NotAssigned = 1,
  /** 未開始 */
  NotStarted = 2,
  /** 如期 */
  OnSchedule = 3,
  /** 注意 */
  AtRisk = 4,
  /** 延遲 */
  Delayed = 5,
  /** 已完成 */
  Completed = 6,
}
